recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
```

Comment:
We wanted to write recipes that would work on any system without requiring to install anything.
I mean we didn't want to assume that `gcc` is installed.